tools/tfrecord2idx

```diff
@@ -24,7 +24,7 @@ while True:
         # crc
         f.read(4)
         idx.write(str(current) + ' ' + str(f.tell() - current) + '\n')
-    except:
+    except Exception:
```

Comment:
why?

Reply:
https://lgtm.com/rules/6780080/

Reply:
It doesn't make any difference here but it sounds like a good practice.